index.html

```diff
@@ -67,6 +70,37 @@
       </form>
     </script>
 
+    <script type="text/ng-template" id="setupAreaDamageForm.html">
+      <form name="areaDamageForm" id="areaDamageForm">
+        Health change (negative if healed):
+        <input type="number" data-ng-model="areaDamage" onFocus="this.select();" autofocus>
+
+        <div class="scrollable medium-height">
+          <table id="charactersInAreaDamage" style="width: 100%">
```

Comment:
Could you please move this style to the CSS?

Otherwise it looks good to go in!